nodejs/eks/cluster.ts

```diff
@@ -507,6 +507,11 @@ export class Cluster extends pulumi.ComponentResource {
      */
     public readonly eksClusterIngressRule: aws.ec2.SecurityGroupRule;
 
+    /**
+     * The default Node Group configuration, or undefined if `skipDefaultNodeGroup` was specified.
+     */
+    public readonly defaultNodeGroup: NodeGroupData | undefined;
```

Comment:
That's right - and it will always be the same security group in this case.  I think that's okay, both because it doesn't seem necessary to break either of these existing interfaces to remove it, and because both sides of this can in certain cases create the security group themselves, so there isn't a clear single place to expose this.  It just happens in the `defaultNodeGroup` case the security group will always match.